bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -105,9 +105,10 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
 
         BagsManager.addBagsStateListener((key, oldStatus, newStatus) -> {
-            if (key.equals(getContextObservation().getNodeKey()))
+            if (Optional.ofNullable(getContextObservation()).map(co -> key.equals(co.getNodeKey())).orElse(false)) {
```

Comment:
(I ask partly because it seems like the lack of a good `exists` makes for uglier code. For example, this could have been `ImOption.apply(getContextObservation()).exists(co -> key.equals(co.getNodeKey())`.